recipes/ignition-tools/all/conanfile.py

```diff
@@ -64,8 +71,12 @@ def validate(self):
                     )
                 )
 
+    def requirements(self):
+        self.requires("ruby/3.1.0")
```

Comment:
There's no reference in the code to ruby 🤔 the only thing I found was they removed it as a build dep https://github.com/gazebosim/gz-tools/blob/20bcaf96bed27850b256fe3ac95c0aa1640143fe/Changelog.md#ignition-tools-130-2021-09-27

Reply:
that's a mistake from my end. We created recipes for version 1.0.0 in our organization in which ruby is needed and I copied the recipe to CCI. Will remove it. Thanks for pointing out and for the thorough review